tests/base.py

```diff
@@ -34,3 +36,24 @@ def get_obj_inventario(self, array_yaml):
                 vm_redes
             )
         return inventario
+
+    def get_dados_ansible_vm(self, array_yaml, nome_vm):
+        lista_dados_ansible = {}
+
+        for maquina_virtual in array_yaml[0][0]['vms']:
+            if maquina_virtual.get('nome') != nome_vm:
+                continue
+
+            if maquina_virtual.get('ansible'):
+                for item in maquina_virtual.get('ansible'):
+                    grupo = item.get('grupo')
+                    dados_ansible = VMAnsible(grupo)
+
+                    if item.get('vars'):
+                        for variavel in item.get('vars'):
```

Comment:
Não seria possível evitar esse if utilizando o retorno padrão do get?

Ex: for variavel in item.get('vars', []):

Reply:
Top. Faz sentido. Vou alterar.